### PR TITLE
french logo to /_promo

### LINKED ISSUE #850: French sticker logo
for comment. I tried a cedilla accent but it didn't work. Looked hindi.
programminghistorianfrgreen

### PR DESCRIPTION
**Pull request by acrymble**
The most popular logo option for the French project. Closes #850

### CODE REVIEW

**Comment by spapastamkou** [comment]
thank you, Adam, it is a good start:) Could help to chose the name of the FR edition as well.

**Comment by acrymble** [comment]
@spapastamkou thanks. Can you click on the "review" button? I can't merge this until you approve.

**Review by spapastamkou — APPROVED** [review]

**Pull request merged**
